### PR TITLE
elementary Gtk3 theme

### PR DESCRIPTION
**Pull request by MrMarukesu**
elementaryOS Gtk Theme for Flatpak apps

### CODE REVIEW

File: org.gtk.Gtk3theme.elementary.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
+  <id>org.gtk.Gtk3theme.elementary</id>
+  <metadata_license>GPL-3.0</metadata_license>
```
Comment (TingPing) [inline org.gtk.Gtk3theme.elementary.appdata.xml]: Perhaps you are confusing this with `<project_license>`, generally people put the metadata (the contents of this file) into the public domain with `<metadata_license>CC0-1.0</metadata_license>`

**Comment by MrMarukesu** [comment]
Oops, I forgot that. adjusted.

**Comment by TingPing** [comment]
bot, build org.gtk.Gtk3theme.elementary

**Comment by TingPing** [comment]
Repository has been created: https://github.com/flathub/org.gtk.Gtk3theme.elementary

**Pull request closed without merging**